src/js/player.js

```diff
@@ -1900,6 +1900,18 @@ class Player extends Component {
   }
 
   /**
+   * Reset the player. Removes all sources, loads the Html tech, and calls `load`.
+   *
+   * @return {Player} Returns the player
+   * @method reset
+   */
+  reset() {
+    this.loadTech_('Html5', null);
```

Comment:
Switch to the Html5 tech. Passing in `null` for `source` in `loadTech_` would load the tech in ["late init" mode](https://github.com/gkatsev/video.js/blob/reset/src/js/tech/html5.js#L33-L41) if it needed to switch from another tech.